src/Server/Controller.py

```diff
@@ -0,0 +1,74 @@
+import sys
+import os
+import json
+project_root =sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+from Utils.Logger import log_activity
+project_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.append(project_root)
+from Models.Notification import Notification
+from Models.UserLogin import User
+from User_Services import User_Service
+from Database.SQLConnect import DatabaseConnection
+
+class ClientHandler:
+    def __init__(self, client_socket):
+        self.client_socket = client_socket
+        self.notification = Notification()
+        self.connection=DatabaseConnection().get_connection()
+        self.user_service = User_Service(self.connection)
+
+    def handle_client(self):
+        while True:
+            try:
+                request = self.client_socket.recv(1024).decode('utf-8')
+                if not request:
+                    break
+                print(f"Received request: {request}")
+                response = self.process_request(request)
+                self.client_socket.send(response.encode('utf-8'))
+            except ConnectionResetError:
+                break
+            except Exception as e:
+                print(f"Error: {e}")
+                break
+        self.client_socket.close()
+
+    def process_request(self, request):
+        request_parts = request.split(",")
+        if request_parts[0] == "verify":
+            return self.verify_user(request_parts[1], request_parts[2])
+        else:
+            return self.handle_role_based_request(request_parts)
+
+    def verify_user(self, username, password):
+        role_name, employee_id = User.verify_employee(username,password,self.connection)
```

Comment:
here you are passing username, password and connection for verify_employee function. but the actual verify_employee has different parameter names. 
Please correct those and check the same in all the place in the code.